soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4431,8 +4431,8 @@ void Randomizer::CreateCustomMessages() {
     // RANDTODO: Translate into french and german and replace GIMESSAGE_UNTRANSLATED
     // with GIMESSAGE(getItemID, itemID, english, german, french).
     const std::vector<GetItemMessage> getItemMessages = {
-        GIMESSAGE(RG_ICE_TRAP, ITEM_NONE, "\x08\x06\x30You are a %bFOWL%w!\x0E\x20",
-                  "\x08\x06\x15 Du bist ein %bDUMMKOPF%w!\x0E\x20", "\x08\x06\x50%bIDIOT%w\x0E\x20"),
+        GIMESSAGE(RG_ICE_TRAP, ITEM_NONE, "\x08\x06\x30You are a %bFOWL%w!",
+                  "\x08\x06\x15 Du bist ein %bDUMMKOPF%w!", "\x08\x06\x50%bIDIOT%w"),
```

Comment:
looks like `0E` is "End With Fade" and `20` is "Space"

were these causing issues?

Reply:
I'm pretty those are for dismissing the box automatically after a certain number of frames, followed by said number of frames. Before now that message was only present for Skulltulas where the auto-dismiss was wanted, but now we're using it for more stuff. Later on I'll be using the ITEM_FROM_SKULLTULA enum value to dynamically add the auto dismiss to every item that comes from a skulltula.